packages/animations/lib/src/page_transition_switcher.dart

```diff
@@ -139,20 +156,28 @@ typedef PageTransitionSwitcherTransitionBuilder = Widget Function(
 /// progress indicator with key A again, all in rapid succession, then the old
 /// progress indicator and the image will be fading out while a new progress
 /// indicator is fading in.
+///
+/// PageTransitionSwitcher uses the property [layoutBuilder] to lay out all
+/// active entries.
+/// If the layout of the transitioning child widgets are not what you are looking
+/// for, see [PageTransitionSwitcher.layoutBuilder] for suggestions on how to
+/// configure the layout of the incoming and outgoing child widgets
```

Comment:
Here are some grammatical suggestions. I also removed the usage of "active entries" since users will be more familiar with referring to the new and old child widgets based on prior documentation.

```suggestion
/// PageTransitionSwitcher uses the [layoutBuilder] property to lay out the
/// old and new child widgets. By default, [defaultLayoutBuilder] is used. 
/// See the documentation for [layoutBuilder] for suggestions on how to
/// configure the layout of the incoming and outgoing child widgets if
/// [defaultLayoutBuilder] is not your desired layout.
```